packages/components/src/theme/domainproperties.scss

```diff
@@ -819,3 +817,39 @@
 .drag-drop-handle .fa-ellipsis-v {
     margin-right: 2px;
 }
+
+.domain-system-fields {
+    margin-top: 15px;
+    margin-bottom: 25px;
+}
+
+.domain-system-fields-header {
+    display: flex;
+    align-items: center;
+}
+
+.domain-system-fields-header__icon {
+    color: $gray-no-highlight;
+    margin-bottom: 2px;
+    margin-right: 10px;
+    cursor: pointer;
+}
+
+.domain-system-fields-header__text {
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 26px;
+    cursor: pointer;
+}
+
+.domain-system-fields__grid {
+    margin-top: 10px;
+}
+
+.domain-custom-fields-header__text {
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 26px;
+    margin-bottom: 5px;
+    width: 240px;
```

Comment:
curious why this needs a width set?

Reply:
Ah, that was a remnant from when I copy-pasted from an earlier version of .domain-system-fields-header__text—good catch, removed!